job/src/slipstream/job/executor.py

```diff
@@ -35,15 +35,16 @@ def _process_jobs(self, thread_name):
             job_uri = None
             try:
                 job_uri = queue.get()
-                job = Job(self.ss_api, job_uri)
 
-                if job is None:
+                if job_uri is None:
```

Comment:
This check was changed from `job` to `job_uri`, as (I think) `job` can never be `None` from looking at the Job constructor.